pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -715,6 +741,8 @@ def process(metadata_dir, conf, es_index, es, donor_output_jsonl_file, bam_outpu
     # hard-code the file name for now    
     train2_freeze_bams = read_train2_bams('../pcawg-operations/variant_calling/train2-lists/Data_Freeze_Train_2.0_GoogleDocs__2015_04_10_1150.tsv')
 
+    santa_cruz_freeze_entries = read_santa_cruz_entries('santa_cruz_pilot.v2.2015_0504.tsv', 'santa_cruz_freeze_entry.tsv')
+
```

Comment:
the santa_cruz release files are committed to pcawg-operations repo now. can be used the same way as train2-list file